zcash_client_backend/src/fees/common.rs

```diff
@@ -372,7 +372,7 @@ where
                 .map(|_| P2PKH_STANDARD_OUTPUT_SIZE),
         );
 
-    let fee_without_change = cfg
+    let min_fee = cfg
```

Comment:
Please reconsider the wording of the comment on lines 333-356 in light of this change.